ci/environment-py34.yml

```diff
@@ -3,7 +3,6 @@ channels:
   - conda-forge
 dependencies:
   - python=3.4
-  - scipy=0.18
   - mkl=2017.0.3
```

Comment:
Do you think we can remove this now too?

Reply:
Good call, let's see what that does.

(FYI I might have to run for the night before this finishes.  If so I'll return to it tomorrow.)